core/templates/pages/community-dashboard-page/modal-templates/translation-modal.controller.spec.ts

```diff
@@ -97,7 +98,8 @@ describe('Translation Modal Controller', function() {
     expect($scope.loadingData).toBe(false);
   });
 
-  it('should get suggested text to translated again', function() {
+  it('should get suggested text to translated again from backend when' +
+    ' suggesting translated text', function() {
```

Comment:
I don't really think this is the key criterion that defines this test. What is the defining property that actually causes "getting suggested text again from backend" to be the right thing to do here, but not in other cases of translation suggestions in general?

Also, "when suggesting translated text" is not that clear -- do you mean "when contributor provides a translation" or "when the contributor dashboard suggests text to be translated"?

Reply:
Done.